### PR TITLE
meeting on 2023-02-24

### PR DESCRIPTION
**Pull request by aidnurs**

### CODE REVIEW

**Review by markus2330 — APPROVED** [review]
Thank you again, great job!

File: doc/meetings/2023_02_24.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Attendees:
+- Markus
+- Yvonne
+- Nursultan
+- Samuel
+
+## Meeting notes:
+
+- Nextcloud calendar
+	- we don't know yet how in which format it is stored on the nextcloud instance e.g. as a file or in a database
```
[the remark this answers is not in the document]
Comment (aidnurs) [reply doc/meetings/2023_02_24.md:15]: I think for people who didn't attend the meeting would be better to have an access to the original discussion. Such conclusions should go directly into issues/PRs in my opinion

File: doc/meetings/2023_02_24.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	- networking
+	- find people to share garden with
+- Locations
+  - location of a user when creating a profile
+  - location of a garden when creating one
+- drawing layer 
+  - has no semantical meaning
+  - general approach for users to construct
+  - name of the layer is up to the user
+- Nextcloud use cases @Samuel
```
[the remark this answers is not in the document]
Comment (aidnurs) [reply doc/meetings/2023_02_24.md:38]: For me `@` indicates an assigned person in general. We could also proceed with the way suggested by you

**Review by markus2330 — APPROVED** [review]
LGTM

**Pull request merged**